compiler/frontend/beam/src/main/java/org/apache/nemo/compiler/frontend/beam/transform/CombineFnFinalTransform.java

```diff
@@ -46,6 +46,7 @@
 public final class CombineFnFinalTransform<K, A, O>
   extends NoWatermarkEmitTransform<WindowedValue<KV<K, A>>, WindowedValue<KV<K, O>>> {
   private static final Logger LOG = LoggerFactory.getLogger(CombineFnFinalTransform.class.getName());
+  //NOSONAR
```

Comment:
Could you please add some explanation for suppressing sonarcloud warnings?

Reply:
Dealt with in commit 028eded